skytemple/module/symbols/model_getter.py

```diff
@@ -0,0 +1,156 @@
+#  Copyright 2020-2024 Capypara and the SkyTemple Contributors
+#
+#  This file is part of SkyTemple.
+#
+#  SkyTemple is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  SkyTemple is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with SkyTemple.  If not, see <https://www.gnu.org/licenses/>.
+from typing import Dict, List
+
+from skytemple_files.data.md.protocol import Gender
+
+from skytemple_files.common.types.file_types import FileType
+
+from skytemple.core.rom_project import RomProject
+
+from gi.repository import Gtk
+
+from skytemple.core.string_provider import StringType
+from skytemple_files.hardcoded.symbols.manual.enums import get_enum_values, get_all_enum_types
+from skytemple_files.hardcoded.symbols.unsupported_type_error import UnsupportedTypeError
+
+_instance: "ModelGetter | None" = None
```

Comment:
Ohhhh, so that's how you avoid ImportErrors when referencing a type before or during its declaration. The only workaround I knew was using a string to declare the type. Good to know.

I've changed it in all files that needed it. Why should it be included in the others?

Reply:
I just like to have it everywhere so Python parses the annotations consistently everywhere. Having that future enabled changes the runtime characteristics of annotations.

Reply:
Should be done